src/platform/user/profile/vet360/actions/transactions.js

```diff
@@ -164,10 +164,14 @@ export function createTransaction(
         ? await apiRequest(route, options)
         : await localVet360.createTransaction();
 
-      recordEvent({
-        event: method === 'DELETE' ? 'profile-deleted' : 'profile-transaction',
-        'profile-section': analyticsSectionName,
-      });
+      // We want the validateAddreses method handling dataLayer events for saving / updating addresses.
+      if (!fieldName.toLowerCase().includes('address')) {
```

Comment:
With this change we will no longer have profile form submission events recorded for home or mailing address updates which feels... wrong. Also this change isn't gated by the address validation feature flag so it will immediately have an impact on production GA.

Reply:
This is the dashboard view that will be affected, FWIW https://analytics.google.com/analytics/web/#/report/content-event-events/a50123418w177519031p176188361/explorer-segmentExplorer.segmentId=analytics.eventAction&explorer-table.plotKeys=%5B%5D&explorer-table.rowStart=0&explorer-table.rowCount=25&_r.drilldown=analytics.eventLabel:profile-transaction/

Reply:
> This is the dashboard view that will be affected, FWIW https://analytics.google.com/analytics/web/#/report/content-event-events/a50123418w177519031p176188361/explorer-segmentExplorer.segmentId=analytics.eventAction&explorer-table.plotKeys=%5B%5D&explorer-table.rowStart=0&explorer-table.rowCount=25&_r.drilldown=analytics.eventLabel:profile-transaction/

It still has the form submission as its handled in the AddressValidationModal component in its submit handler.  I will gate this, though.

Reply:
okay thanks I see it now

Reply:
buuuuuut what about the case where the address validation modal is skipped entirely, when will this event be sent to GA?